app/views/welcome/index.html.erb

```diff
@@ -33,11 +33,27 @@
           <label class="form-label" for="AB_select">
             A/B test being used
           </label>
-          <select class="form-control" name="which_test" id="AB_select">
+          <select class="form-control" name="which_test" id="AB_select" onchange="showForm()">
```

Comment:
I believe the preferred option is to add the event listener in JS rather than in the html.

Reply:
done!